Makefile

```diff
@@ -16,7 +16,8 @@ IMG_TAG := latest
 endif
 
 ifeq (${LOCAL_PROVIDER_VERSION},latest)
-LOCAL_PROVIDER_VERSION := v0.0.0
+# Change this versions after release when required here and in e2e config (test/e2e/config/nutanix.yaml)
+LOCAL_PROVIDER_VERSION := v1.3.99
```

Comment:
We've had good success with https://github.com/caarlos0/svu and then  `svu patch --pre-release dev` to get whatever the next dev version is.

I suspect this will get stale pretty quickly, but we can do a follow up PR.

Reply:
Oh I see the same version needs to be used in test/e2e/config/nutanix.yaml

Reply:
@dkoshkin Good thing, but I think it should be done in separate PR with improvement of test scripts (including generation of e2e config). We already discuss it with @jimmidyson.